src/accelerate/utils.py

```diff
@@ -738,6 +739,68 @@ def __post_init__(self):
         }
 
 
+@dataclass
+class FullyShardedDataParallelPlugin:
+    """
+    This plugin is used to enable fully sharded data parallelism.
+    """
+
+    from torch.distributed.fsdp.fully_sharded_data_parallel import BackwardPrefetch, CPUOffload, ShardingStrategy
```

Comment:
As said offline this is the line making the CI red.

Reply:
Got it. Thanks for pointing this out. Fixed this.